src/sandstorm/grain.capnp

```diff
@@ -528,7 +528,12 @@ struct SturdyRefOwner {
   # Human-readable explanation of how this capability relates to the grain. This is displayed
   # when visualizing outgoing capabilities from the calling grain in the Sandstorm UI. This string
   # will NOT be communicated to the host of the saved capability; it is intercepted and used by the
-  # Sansdtorm system.
+  # Sandstorm system.
+
+  userId @1 :Data;
+  # ID of the user whose permissions are tied to this capability. This data will NOT be communicated
```

Comment:
Yeah, I couldn't see where else the `userId` was going to come from. It sounds like the PowerBox is going to be able to handle this, which is cool.